protocol-units/bridge/contracts/src/AtomicBridgeInitiator.sol

```diff
@@ -31,6 +31,9 @@ contract AtomicBridgeInitiator is IAtomicBridgeInitiator, OwnableUpgradeable {
     IWETH9 public weth;
     uint256 private nonce;
 
+    // Constant time lock of 24 hours in seconds (86400 seconds)
+    uint256 public constant TIME_LOCK_DURATION = 24 * 60 * 60;
```

Comment:
What if you name this constant INITIATOR_TIME_LOCK_DURATION and do the multiplication by 2 here to handle the multiplication all in one place and name the constant so that it's distinct from COUNTERPARTY_TIME_LOCK_DURATION. Just maybe a little clearer for readers and avoids the need to multiply by 2 in various places throughout the modules / contracts.